backends/advanced/src/advanced_omi_backend/routers/modules/health_routes.py

```
@@ -92,6 +92,20 @@ async def auth_health_check():
 @router.get("/health")
 async def health_check():
     """Comprehensive health check for all services."""
+    # Load model config once for display fields
+    try:
+        _cfg = _load_root_config() or {}
```

Ruff (0.14.10)

Undefined name `_load_root_config`

(F821)